<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow</class>
 <widget class="QWidget" name="userWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户中心</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="userLabel">
      <property name="text">
       <string>欢迎您，</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="user_merchant">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>901</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="user">
    <attribute name="title">
     <string>个人账户信息</string>
    </attribute>
    <widget class="QGroupBox" name="passwordPart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>密码修改</string>
     </property>
     <widget class="QLineEdit" name="password1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="password2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>输入新密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>再次输入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeConfirm">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="moneyPart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>账户余额</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>当前余额(元）</string>
      </property>
     </widget>
     <widget class="QLabel" name="money">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="topUpComfirm">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>确认充值</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入充值金额(元)</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="topUpNumber">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <double>2500.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>641</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string>我的购物车</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>我的订单</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="merchants">
    <attribute name="title">
     <string>商户信息</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>881</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>我的商品</string>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>统一设置折扣(%)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="clothDis">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>服装</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>食品</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="foodDis">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>图书</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="bookDis">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="pageSum">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>410</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>共计 1 页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lastPage">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
     <widget class="QLabel" name="pageNow">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>410</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>当前第 页</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPage">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="clothTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>871</width>
        <height>331</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>106</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>42</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>42</number>
      </attribute>
      <row>
       <property name="text">
        <string>line1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>line7</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>商品名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>商品类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>商品描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>更多</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>原价</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>剩余数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>折扣</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>操作</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="1" column="7">
       <property name="text">
        <string>   </string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="2" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="2" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="2" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="3" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="4" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="5" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string> </string>
       </property>
      </item>
      <item row="6" column="2">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="3">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="4">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="5">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="6">
       <property name="text">
        <string>  </string>
       </property>
      </item>
      <item row="6" column="7">
       <property name="text">
        <string>  </string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>确认设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>添加商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_1">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>100</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_2">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>140</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_3">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>180</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_4">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>220</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_5">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>260</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_6">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>300</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit_7">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>340</y>
        <width>89</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>user_merchant</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
